docs/cloud/tutorials/integrating-external-services.md

```diff
@@ -0,0 +1,114 @@
+# Integrating External Services
+
+Kubernetes, and by extension OpenShift OKd, gives a lot of flexibility regarding network use cases. One of the uses cases is the one that allows to use external services, like databases, transparently in a OpenShift project. Other of the use cases would be to to have a network proxy between OpenShift installations. This could be used as a temporal to ease migration periods.
```

Comment:
```suggestion
Kubernetes, and by extension OpenShift OKd, gives a lot of flexibility regarding network use cases. One of the uses cases is the one that allows to use external services, like databases, transparently in a OpenShift project. Other of the use cases would be to to have a network proxy between OpenShift installations. This could be used as a temporal solution to ease migration periods.
```